src/extras/PMREMGenerator.js

```diff
@@ -753,7 +751,7 @@ function _getBlurShader( lodMax, width, height ) {
 
 function _getEquirectShader() {
 
-	const shaderMaterial = new RawShaderMaterial( {
+	const shaderMaterial = new ShaderMaterial( {
```

Comment:
`_getEquirectShader()` does not return a shader. It returns a material.

`_getCommonVertexShader()` returns a shader.

Reply:
Fixed: https://github.com/mrdoob/three.js/commit/28ce323f825a7f92587a84838c42d25beb2e7d10

Reply:
@mrdoob  likewise `_getCubemapShader()`

Reply:
Fixed too: https://github.com/mrdoob/three.js/commit/1db78b84b0152fcd561314140a6751109c894906